mlflow/server/handlers.py

```diff
@@ -159,9 +159,6 @@ def _get_experiment():
     response_message = GetExperiment.Response()
     experiment = _get_store().get_experiment(request_message.experiment_id).to_proto()
     response_message.experiment.MergeFrom(experiment)
-    run_info_entities = _get_store().list_run_infos(request_message.experiment_id,
```

Comment:
Does this account for change in expected outcome? Would there be users that depend on runs returned with experiment?

Reply:
As discussed offline, this field has been deprecated as of MLflow 1.2 and is not consistently populated across backend stores. The Search API is now the recommended mechanism for fetching runs. For these reasons, @mparkhe and I have agreed that depopulating here makes sense.